### ISSUE TITLE
All extensions marked as "possibly incompatible" for notebook 5.0.0

### ISSUE THREAD

**Issue by andyneff** [ipython-contrib/jupyter_contrib_nbextensions#942]
I'm guessing the compatibility in the yaml files need to be updated to include 5.x (for those that work in 5.x ;) )

**Comment by jcb91** [comment]
Yes, good point. I think quite a few have been used successfully on 5.0 (see https://github.com/jupyter/notebook/issues/2151 for details), but we do still need to update all the yaml files...

**Comment by jfbercher** [comment]
> but we do still need to update all the yaml files...

Oops :-(

**Comment by jcb91** [comment]
So, here's my list of what we need to update. If anyon'es tried any of these in
5.0 and found them to work (or not!), please leave a comment below.
I'll start a PR to update the yaml files which you can also PR updates to
directly.

The following need compatibility updating:
(a filled checkbox means compatible, but not yet declared as such)

 [ ] addbefore/main
 [ ] autosavetime/main
 [ ] autoscroll/main
 [ ] chrome-clipboard/main
 [ ] code_font_size/code_font_size
 [ ] code_prettify/2to3
 [ ] code_prettify/autopep8
 [ ] code_prettify/code_prettify
 [ ] codefolding/edit
 [ ] codefolding/main
 [ ] codemirror_mode_extensions/main
 [ ] contrib_nbextensions_help_item/main
 [ ] datestamper/main
 [ ] dragdrop/main
 [ ] equation-numbering/main
 [ ] execute_time/ExecuteTime
 [x] exercise/main
 [x] exercise2/main
 [ ] gist_it/main
 [ ] help_panel/help_panel
 [ ] hide_header/main
 [ ] hide_input/main
 [ ] hide_input_all/main
 [x] highlighter/highlighter
 [x] hinterland/hinterland
 [ ] init_cell/main
 [ ] keyboard_shortcut_editor/main
 [ ] limit_output/main
 [ ] move_selected_cells/main
 [ ] navigation-hotkeys/main
 [ ] nbTranslate/main
 [ ] notify/notify
 [ ] printview/main
 [x] python-markdown/main
 [ ] qtconsole/qtconsole
 [ ] rubberband/main
 [ ] ruler/main
 [ ] runtools/main
 [ ] scratchpad/main
 [ ] scroll_down/main
 [ ] search-replace/main
 [x] select_keymap/main
 [ ] skill/main
 [ ] skip-traceback/main
 [ ] snippets/main
 [ ] snippets_menu/main
 [ ] spellchecker/main
 [ ] splitcell/splitcell
 [ ] table_beautifier/main
 [x] toc2/main
 [ ] toggle_all_line_numbers/main
 [ ] tree-filter/index
 [ ] zenmode/main


already marked as compatible:

 [x] collapsible_headings/main
 [x] comment-uncomment/main
 [x] freeze/main
 [x] highlight_selected_word/main
 [x] latex_envs/latex_envs

@jfbercher, I think you mentioned in https://github.com/jupyter/notebook/issues/2151 that you'd tested hide_menubar -  I guess you mean hide_header?

**Comment by jfbercher** [comment]
> @jfbercher, I think you mentioned in jupyter/notebook#2151 that you'd tested hide_menubar - I guess you mean hide_header?

probably...

**Comment by baldwint** [comment]
I tried `notify/notify` in 5.x and it seems to work just fine without any modification.

**Comment by hrogal** [comment]
I just used spellchecker with 5.0.0 and it worked fine.

**Comment by jfbercher** [comment]
:+1: @hrogal 
In coming days, I will try to make a big tour of available extensions and update the relevant yaml files.

**Comment by elmerehbi** [comment]
I cannot enable zenmode. It says `zenmode/main) is referenced by two different yaml files on the server`

**Comment by jcb91** [comment]
@elmerehbi it sounds like you have two separate issues. 

> It says zenmode/main) is referenced by two different yaml files on the server

This sounds like you have two different installations in different places (e.g. a per-user and a system-wide installation). Check the notebook server logs for further details.

> Actually it is enabled but doesn't work. I cannot find the toolbar icon that I used before.

Not sure about this, but it's probably best to open a separate issue, and include anything from the JavaScript console...

**Comment by elmerehbi** [comment]
@jcb91 
`This sounds like you have two different installations in different places (e.g. a per-user and a system-wide installation). Check the notebook server logs for further details.`

What do you mean by two installations; Python or the notebook extensions? As far as I remember I installed Anaconda then created an environment in which I installed my modules.

**Comment by jcb91** [comment]
> What do you mean by two installations; Python or the notebook extensions? As far as I remember I installed Anaconda then created an environment in which I installed my modules.

I mean that the nbextension files exist in two places in the jupyter path. This is distinct from the python/conda package. The anaconda packages automatically install the nbextension files to the sys-prefix location, so if you then run something like `jupyter contribute nbextensions install --user`, you'd end up with a second copy in the user directories. To find exactly where they are, check server logs (not the browser console logs, but output from the `jupyter notebook` or similar command that you use to start the server) for lines from the configurator about duplicates. 

For the second part, see reply in the new issue :)

**Comment by jcb91** [comment]
You may need to run the server with the debug flag to see the appropriate log messages though. See http://jupyter-contrib-nbextensions.readthedocs.io/en/latest/troubleshooting.html#removing for more information.

**Comment by elmerehbi** [comment]
> if you then run something like jupyter contribute nbextensions install --user

Never ran such a command as far I remember.

Thanks for the link.
